packages/rum-core/src/domain/error/trackConsoleError.ts

```diff
@@ -1,18 +1,12 @@
 import type { Observable, RawError } from '@datadog/browser-core'
-import { clocksNow, ErrorHandling, initConsoleObservable, ErrorSource, ConsoleApiName } from '@datadog/browser-core'
+import { initConsoleObservable, ConsoleApiName } from '@datadog/browser-core'
 
 export function trackConsoleError(errorObservable: Observable<RawError>) {
-  const subscription = initConsoleObservable([ConsoleApiName.error]).subscribe((consoleError) =>
-    errorObservable.notify({
-      startClocks: clocksNow(),
-      message: consoleError.message,
-      stack: consoleError.stack,
-      fingerprint: consoleError.fingerprint,
-      source: ErrorSource.CONSOLE,
-      handling: ErrorHandling.HANDLED,
-      handlingStack: consoleError.handlingStack,
-    })
-  )
+  const subscription = initConsoleObservable([ConsoleApiName.error]).subscribe((consoleLog) => {
+    if (consoleLog.error) {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Is this condition necessary as we init the console observable with `ConsoleApiName.error`?